spec/function.dd

```diff
@@ -2015,6 +2015,48 @@ S test6(ref return scope int* p)
 }
 ---
 
+$(H4 $(LNAME2 pure-scope-inference, Inferred `scope` parameters in `pure` functions))
```

Comment:
I think this should be H3 as it's not a subheading of 'return ref scope'.
```suggestion
$(H3 $(LNAME2 pure-scope-inference, Inferred `scope` parameters in `pure` functions))
```

Reply:
Good catch. I haven't seen this rendered yet because my local build of dlang.org is suddenly acting up with linking errors.

Reply:
Thanks. Just FYI you can click the Details link on the CyberShadow test, then click New for function.html to see the changes rendered.

Reply:
I didn't know that, very useful.